stacks/fdo/docker-stack.yml

```diff
@@ -22,8 +22,7 @@ services:
       log4j_configuration_file: log4j2.xml
     volumes:
       - rv-target:/home/fdo/target
-    networks:
-      - fdonetwork
+    network_mode: host
```

Comment:
Host networking?

Reply:
Yes, it was set up this way upstream and it makes it hard for the Owner to find the RV otherwise.